src/custom/state/sentry/updater/index.ts

```diff
@@ -0,0 +1,53 @@
+import { useEffect } from 'react'
+import * as Sentry from '@sentry/browser'
+
+import useIsWindowVisible from 'hooks/useIsWindowVisible'
+import { useSwapState } from 'state/swap/hooks'
+import { SupportedChainId } from 'constants/chains'
+import { useCurrency } from 'hooks/Tokens'
+import { useWalletInfo } from 'hooks/useWalletInfo'
+
+export default function Updater(): null {
+  const { account, chainId, walletName, isSupportedWallet } = useWalletInfo()
+  const windowVisible = useIsWindowVisible()
+
+  const {
+    INPUT: { currencyId: sellTokenAddress },
+    OUTPUT: { currencyId: buyTokenAddress },
+  } = useSwapState()
+
+  const sellCurrency = useCurrency(sellTokenAddress)
+  const buyCurrency = useCurrency(buyTokenAddress)
+
+  // create sentry context based on "main" parameters
+  useEffect(() => {
+    if (windowVisible) {
+      Sentry.setContext('user', {
+        userAddress: account || 'DISCONNECTED',
+        wallet: walletName,
+        isWalletSupported: isSupportedWallet,
```

Comment:
I wouldn't add this, because adds noise (one more field), and i don't see it adds too much value, since its derived data anyways.

<img width="827" alt="image" src="https://user-images.githubusercontent.com/2352112/162180219-09c20ddd-d899-4766-92ac-72e62bba6bde.png">